### ISSUE TITLE
`Object.keys` considered a component

### ISSUE THREAD

**Issue by CodeSmith32** [ArnaudBarre/eslint-plugin-react-refresh#93]
We were using this plugin, but it appeared that `Object.keys` was considered a 'component' somehow, and raised a warning in a file consisting of solely constants and constant logic. e.g.

```ts
// A type-safe way to maintain a list of enum values as a javascript array:
export const TABLE: Record<EnumType, true> = {
  key1: true,
  key2: true,
  key3: true,
}
export const ENUM_VALUES = Object.keys(TABLE) as EnumType[];
```

Because the first letter of the constant name, `ENUM_VALUES` is capital, it *could* be a component. I'm not sure what the parser looks at after that, but apparently `Object.keys(...)` looks too much like a component, when it most definitely isn't.

**Comment by ArnaudBarre** [comment]
Thanks I will look into it.

Function call that wrap something that also could be a component is considered maybe a component because that's a pattern in older react/redux codebases. But I think using `_` in the name should disqualify as a component or maybe using a member access (X.Y) should not be considered a potential HOC

**Comment by ArnaudBarre** [comment]
If you have time, can should check if the alpha works for you?

- `npm i eslint-plugin-react-refresh@0.5.0-alpha.0`
- Changelog: https://github.com/ArnaudBarre/eslint-plugin-react-refresh/pull/97/files?short_path=06572a9#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed
